src/state/remote.go

```diff
@@ -22,14 +22,11 @@ func (s *State) doLoadRemoteState() {
 			for _, b := range *response.Result().(*[]*model.Branch) {
 				branch := b
 
-				// Skip ignored branches
-				if !s.manifest.IsBranchAllowed(branch) {
+				// Save to state, skip configs/rows if branch is ignored
+				if s.SetRemoteState(branch) == nil {
```

Comment:
Podmienku som presunul do `SetRemoteState`, aby to bolo pre vsetky objekty.
V pripade nejakej chyby -> `nil` -> a child objekty sa preskocia.